tests/unit/test_synthDrivers/test_espeak.py

```diff
@@ -0,0 +1,83 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2022 NV Access Limited.
+
+"""Unit tests for the eSpeak synth driver submodule.
+"""
+
+import logging
+import unittest
+import logHandler
+from speech.commands import LangChangeCommand
+from source.synthDrivers.espeak import SynthDriver
+
+
+class FakeESpeakSynthDriver:
+	_language = "default"
+	_defaultLangToLocale = {"default": "en-gb"}
+	availableLanguages = {"fr", "fr-fr", "en-gb", "ta-ta"}

```

Comment:
- What does `fr` mean when there is `fr-fr`? 
- Is this representative of the `availableLanguages` that come out of espeak?

Reply:
These are fictional examples used for testing. In eSpeak "fr" doesn't exist. A comparable example, supported in the version of eSpeak we use for 2022.1, would be "pt" and "pt-br".